src/frontend/src/invites.tsx

```diff
@@ -53,6 +124,22 @@ export const Invites = () => {
                             setCredits(parseInt(event.target.value))
                         }
                     />
+                    Per user:
+                    <input
+                        type="number"
+                        value={credits_per_user}
+                        className="max_width_col"
+                        onChange={(event) =>
+                            setCreditsPerUser(parseInt(event.target.value))
+                        }
+                    />
+                    Realm:
+                    <input
+                        type="text"
+                        value={inviteRealm}
+                        className="max_width_col"
+                        onChange={(event) => setInviteRealm(event.target.value)}
+                    />
                     {!busy && (
```

Comment:
I don't think we need this. Check out `ButtonWithLoading` instead. It simply takes an async function and displays loading while the function is called.